.github/workflows/deployment.yml

```diff
@@ -143,7 +143,7 @@ jobs:
         shell: bash
         run: |
           oc login "${{ secrets.OPENSHIFT_CLUSTER }}" --token="${{ secrets.OC4_DEV_TOKEN }}"
-          MODULE_NAME=api SECOND_LEVEL_DOMAIN="apps.silver.devops.gov.bc.ca" VANITY_DOMAIN="${SUFFIX}-dev-psu.apps.silver.devops.gov.bc.ca" bash openshift/scripts/oc_deploy.sh ${SUFFIX} apply
+          MODULE_NAME=api SECOND_LEVEL_DOMAIN="apps.silver.devops.gov.bc.ca" VANITY_DOMAIN="${SUFFIX}-dev-psu.apps.silver.devops.gov.bc.ca" USE_WFWX="True" bash openshift/scripts/oc_deploy.sh ${SUFFIX} apply
```

Comment:
Do we need `USE_WFWX="True"` anymore?

Reply:
Yes, still needed for deployment of our `wps-api` containers.

Reply:
Yeah looks like we still have a lot of flags in the codebase relying on it. I wonder if we should add a dev task to remove them, or keep them around as a fallback? Not sure.

Reply:
Created a ticket to address this: https://github.com/bcgov/wps/issues/2865